src/queries/enum_repr_int_changed.ron

```diff
@@ -18,8 +18,14 @@ SemverQuery(
                         name @tag @output
 
                         attribute {
-                            attr_value: value @filter(op: "regex", value: ["$repr_regex"])
-                                                @tag @output(name: "old_attr")
+                            as_string @output(name: "old_attr")
```

Comment:
(no need for change, just informational) an equivalent way to write this is `old_attr: as_string @output`. I have no preference between the two, whatever works best for you.

Reply:
I changed it as you have shown here - in my opinion it is more clear to have a notation reminiscent to `variable := value`.